treeherder/model/migrations/0040_remove_commit_search_vector_idx_commit_search_vector_and_more_squashed_0041_update_search_vector.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 5.1.5 on 2025-03-06 14:49
+
+import django.contrib.postgres.indexes
+import django.contrib.postgres.search
+from django.contrib.postgres.search import SearchVector
+from django.db import migrations
+
+
+def update_search_vector(apps, schema_editor):
+    Commit = apps.get_model("model", "Commit")
+    Commit.objects.update(
+        search_vector=SearchVector("revision", "author", "comments", config="english")
+    )
+
+
+class Migration(migrations.Migration):
+
+    replaces = [
+        ("model", "0040_remove_commit_search_vector_idx_commit_search_vector_and_more"),
```

Comment:
These two migrations don't exist (outside your local database if it hasn't been reset). `replaces` might even break the migration of the ones mentioned here are missing.

Reply:
Okay will take `replaces` out